src/picosvg/svg_reuse.py

```diff
@@ -14,40 +14,60 @@
 
 # Functions meant to help discern if shapes are the same or not.
 
+import copy
 import dataclasses
+from itertools import islice
+from math import atan2
+from picosvg.geometric_types import Vector, almost_equal
 from picosvg.svg_types import SVGShape, SVGPath
-from typing import Optional, Tuple
+from typing import Generator, Iterable, Optional, Tuple
+from picosvg import svg_meta
 from picosvg.svg_transform import Affine2D
 
 
 # Number of decimal digits to round floats when normalizing or comparing
-DEFAULT_TOLERANCE = 9
+# TODO: hard to get #s to line up well with high tolerance
+# TODO: maybe the input svgs should have higher precision? - only 2 decimals on hearts
+_DEFAULT_TOLERANCE = 6
```

Comment:
This corresponds to an _absolute_ tolerance between almost equal numbers. I wonder if we should maybe make this tolerance _relative_ to the size of the viewBox or UPEM? Two given coordinates may be more or less "almost equal" depending on the size of the reference coordinate space.

Reply:
The caller can achieve this by providing their own tolerance ... maybe I should take a direct value max_err or some such.

Reply:
Tolerance now passed in